Exiled.API/Features/Player.cs

```diff
@@ -1083,6 +1083,11 @@ public bool IsInPocketDimension
         /// </summary>
         public bool IsUsingStamina { get; set; } = true;
 
+        /// <summary>
+        /// Gets or sets a value indicating whether or not the player invisibility effect should not be removed when interacting.
+        /// </summary>
+        public bool KeepInvisibilityOnInteracting { get; set; } = false;
```

Comment:
Just `Gets or sets a value indicating whether the invisibility effect should be removed when interacting.`